packages/components/src/styles/components/advanced-table.scss

```diff
@@ -261,6 +261,7 @@ $hds-advanced-table-button-size: 24px; // the size of the buttons and dropdown t
 // multi-select (isSelectable=true)
 .hds-advanced-table__th-content {
   display: flex;
+  flex-grow: 1;
```

Comment:
For reference: there is something not working when the `Th` is inside the `tbody`, spoke with @shleewhite and she will look into it

<img width="927" height="184" alt="screenshot_5267" src="https://github.com/user-attachments/assets/ca2c0744-4234-49a1-8958-5aaed34388ff" />